src/lib.rs

```diff
@@ -44,15 +47,18 @@ fn most_used(indents: &HashMap<isize, Usage>) -> usize {
     let mut max_used = 0;
     let mut max_weight = 0;
 
-    for (&key, ref usage) in indents.iter() {
+    for (&key, usage) in indents.iter() {
```

Comment:
@chriskrycho I'm sure this is right, but could you share the TL;DR?

Reply:
Very short version: Rust 2018 made this "just work" correctly by way of improving what Rust infers about usage like this.

Reply:
@chriskrycho oh ok that makes sense. If memory serves I was confused at the time that this wasn't inferred. Glad to hear that this has been improved!

Reply:
Err, looks like it landed *independent* of Rust 2018, but was around that time frame.